jaeger_client/reporter.py

```diff
@@ -221,6 +222,18 @@ def _flush(self):
         yield self.queue.join()
 
 
+class ReporterMetrics:
+    def __init__(self, metrics_factory):
+        self.reporter_success = \
+            metrics_factory.counter(name='jaeger.spans', tags={'reported': 'True'})
+        self.reporter_failure = \
+            metrics_factory.counter(name='jaeger.spans', tags={'reported': 'False'})
+        self.reporter_dropped = \
+            metrics_factory.counter(name='jaeger.spans', tags={'dropped': 'True'})
```

Comment:
is this naming pattern consistent with Go?  It looks weird. E.g. there's never going to be dropped=false, so wouldn't a tag-less name `jaeger.spans.reporter-dropped` or such be more appropriate?

I would actually suggest starting a s/s and pulling all the names across 4 clients into it, and then coming up with a single consistent naming scheme (it can be a separate diff from this, if we don't release this one).